src/gui/widgets/assembly.cc

```diff
@@ -538,25 +538,25 @@ settings, otherwise debugging features may not work.)");
             const char* section = "UNK";
             if (addr < 0x00800000) {
                 section = "RAM";
-                code = *reinterpret_cast<uint32_t*>(m_memory->m_psxM + addr);
+                code = *reinterpret_cast<uint32_t*>(m_memory->m_wram + addr);
                 if (addr <= 0x007ffff8) {
-                    nextCode = *reinterpret_cast<uint32_t*>(m_memory->m_psxM + addr + 4);
+                    nextCode = *reinterpret_cast<uint32_t*>(m_memory->m_wram + addr + 4);
                 }
                 base = m_ramBase;
             } else if (addr < 0x00810000) {
                 section = "PAR";
                 addr -= 0x00800000;
-                code = *reinterpret_cast<uint32_t*>(m_memory->m_psxP + addr);
+                code = *reinterpret_cast<uint32_t*>(m_memory->m_exp1 + addr);
                 if (addr <= 0x0000fff8) {
```

Comment:
This becomes technically wrong, due to the size change of the exp1 section. But then again, I need to heavily rewrite this code.